### PR TITLE
Update Standalone

### PR DESCRIPTION
**Pull request by KaiyiLiu1234**
Updated standalone and dyncomp model to develop models for node and container level. Updated predict for standalone model to accept a dict containing feature names and corresponding values. Updated extractor_test.py to contain additional examples. Updated train_types to include enums for Label groups instead of only feature groups.

### CODE REVIEW

**Comment by KaiyiLiu1234** [comment]
By pushing this, I should also be able to test on kind cluster. kepler-model-server-base already installed the sklearn dependency. By making a second push, kepler-model-server should then also install the sklearn dependency from kepler-model-server-base (please let me know if this is incorrect assumption). Also, for the next PR which will be pushed today, I plan to update extractor.py to accept the new kepler metrics. The following PR will then be focused on setting up the dyn-comp pipeline to handle multiple models (for package power and dram power). How this is implemented will depend on @sunya-ch implementation of the new pipeline api.

File: server/train/pipelines/XGBoostRegressionCompDynPipeline/pipe.py [hunk 1]
```diff
@@ -2,40 +2,47 @@
...
-        model_class = 'xgboost'
+        self.train_type = train_type
+        self.model_class = 'xgboost'
         self.energy_source = 'rapl'
-        self.energy_components = ['package', 'dram']
-        features = FeatureGroups[FeatureGroup.CgroupOnly]
-        model_output = ModelOutputType.DynComponentPower
-        self.labels = PACKAGE_LABEL
-        super().__init__(model_name, model_class, model_file, features, model_output)
+        self.feature_group = FeatureGroup.CounterOnly
```
Comment (rootfs) [inline server/train/pipelines/XGBoostRegressionCompDynPipeline/pipe.py:26]: also include irq features

Comment (KaiyiLiu1234) [reply server/train/pipelines/XGBoostRegressionCompDynPipeline/pipe.py]: Included. I should note that in the next PR (which involves updating extractor for kepler metrics), IRQ features will be stored in BPF. Moreover, pipeline won't work yet because the kepler metrics have not been updated in the kepler model server. Will finish this today.

**Review by rootfs — APPROVED** [review]
